tests/test_scripts.py

```diff
@@ -1,22 +1,31 @@
 import os
 from entity_resolver.scripts import run
+import json
 
+# class TestArxiv:
+#
+#     def test_dummy(self):
+#         file_path = 'file.txt'
+#         run(['norm-arxiv', '', file_path])
+#         with open(file_path, 'r') as f:
+#             assert f.read() == 'arxiv'
+#         os.remove(file_path)
 
-class TestArxiv:
 
-    def test_dummy(self):
-        file_path = 'file.txt'
-        run(['norm-arxiv', '', file_path])
-        with open(file_path, 'r') as f:
-            assert f.read() == 'arxiv'
-        os.remove(file_path)
+class TestCiteseer:
 
+    # def test_dummy(self):
+    #     file_path = 'file.txt'
+    #     run(['norm-citeseer', '', file_path])
+    #     with open(file_path, 'r') as f:
+    #         assert f.read() == 'citeseer'
+    #     os.remove(file_path)
+    def test_citeseer(self):
+        file_path ='/Users/xinxinhuang/Desktop/EntityResolution_Capstone/data/citeseer/citeseer-mrdm05.dat'
```

Comment:
Use relative path instead of absolute path so that the test scripts can also get executed successfully on others' computers. In this case, I think it should be `../data/citeseer/citeseer-mrdm05.dat`